rendermodules/stack/swap_zs.py

```diff
@@ -0,0 +1,104 @@
+import renderapi
+from rendermodules.module.render_module import RenderModule, RenderModuleException
+from rendermodules.stack.schemas import SwapZsParameters, SwapZsOutput
+import time
+
+example = {
+    "render": {
+        "host": "em-131db",
+        "port": 8080,
+        "owner": "TEM",
+        "project": "17797_1R",
+        "client_scripts": "/allen/aibs/pipeline/image_processing/volume_assembly/render-jars/production/scripts/"
+    },
+    "source_stack": "source_stack",
```

Comment:
this example is a single stack but the code has this a a list.

Reply:
these examples are dummy ones.. will add a real one...